src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -1488,4 +1622,17 @@ private function getExpressionLanguage()
 
         return $this->expressionLanguage;
     }
+
+    private static function export($value)
+    {
+        if (!is_array($value)) {
+            return var_export($value, true);
+        }
+        $code = array();
+        foreach ($value as $k => $v) {
+            $code[] = sprintf('%s => %s', var_export($k, true), self::export($v));
+        }
+
+        return sprintf('array(%s)', implode(', ', $code));
```

Comment:
with one difference: this generates a single line export, while var_export splits arrays on several lines